src/main/java/org/gatorgradle/display/CheckResult.java

```diff
@@ -133,7 +136,11 @@ public int hashCode() {
   public String textReport(boolean includeDiagnostic) {
     if (outcome) {
       return PASS_SYMBOL + INDENT + check;
-    } else {
+    } else if (check.contains("--reach")) {
+      outcome = True;
```

Comment:
This will break the new `report` task -- instead of modifying the outcome, you should ignore results that come from "reach" checks, or handle them in `CommandOutputSummary` separately. Additionally, it makes me feel a bit uncomfortable using just a containment check for `--reach` to determine the kind of check. Maybe we should add a field to the JSON GatorGrader returns specifying the type of check? This would enable other types in the future as well, if desired.